recipes/parmoo/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "parmoo" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/parmoo-{{ version }}.tar.gz
+  sha256: 5a2606665783b63915913f96393876c44c85c93259153842c3482d68a9ea32b1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - tox
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - scipy
+    - pydoe
+
+test:
+  imports:
+    - parmoo
+  commands:
+    - py.test --pyargs parmoo
+  requires:
+    - libensemble
```

Looks like a test is failing due to missing `mpi4py`:
```suggestion
    - libensemble
    - mpi4py
```

---

So this is an optional dependency of libensemble that I am not using for my tests.  It thinks I am trying to use it because libensemble runs cannot be started through pytest, which causes an error.  Instead, I've updated my pytest recipe to only run unit tests (not regression tests), this should solve the problem.  To run the full test script, we'd need to run a special test script that I wrote, contained in the parmoo/tests directory